mlem/utils/module.py

```diff
@@ -266,8 +266,10 @@ def get_package_name(mod: ModuleType) -> str:
     """
     if mod is None:
         raise ValueError("mod must not be None")
-    name = mod.__name__
-    return MODULE_PACKAGE_MAPPING.get(name, name)
+    if isinstance(mod, str):
+        mod = importing.import_module(mod)
```

Comment:
This will force to import module every time. Can you switch to `if not str: mod = mod.__name__`?